src/plugins/intel_gpu/src/graph/impls/ocl_v2/utils/kernel_generator.cpp

```diff
@@ -128,7 +128,18 @@ Arguments KernelGenerator::get_arguments_desc(const RuntimeParams& params) const
         args.push_back({ArgumentDescriptor::Types::SHAPE_INFO, 0});
     }
 
-    for (uint32_t i = 0; i < params.input_layouts.size(); i++) {
+    size_t num_fused_deps = 0;
```

Comment:
nit: what about `num_fused_deps_without_external_input` to clarify?

Reply:
Renamed